_layouts/home.html

```diff
@@ -1,8 +1,10 @@
 ---
 layout: default
+
+
 ---
   {% include header.html %}
   {% include benefits.html %}
   {% include testimonials.html %}
```

Comment:
Maybe we just delete this line for now and when we have actual data, we restore it?

Reply:
Deleted